src/main/java/systems/composable/dropwizard/cassandra/CassandraFactory.java

```diff
@@ -465,6 +486,8 @@ public Cluster build(MetricRegistry metrics, HealthCheckRegistry healthChecks) {
         queryOptions.ifPresent(builder::withQueryOptions);
         socketOptions.ifPresent(builder::withSocketOptions);
         poolingOptions.map(PoolingOptionsFactory::build).ifPresent(builder::withPoolingOptions);
+        translatorOptions.map(AddressTranslatorFactory::build).ifPresent(builder::withAddressTranslator);
+
```

Comment:
Please don't add extra whitespace. The PR should conform to the existing code styles used.